lib/src/authorization_code_grant.dart

```diff
@@ -125,8 +123,8 @@ class AuthorizationCodeGrant {
   /// query parameters provided to the redirect URL.
   ///
   /// It is a [StateError] to call this more than once.
-  Uri getAuthorizationUrl(Uri redirect, {Iterable<String> scopes,
-      String state}) {
+  Uri getAuthorizationUrl(Uri redirect,
+      {Iterable<String> scopes, String state, String delimiter: ' '}) {
```

Comment:
Document the delimiter parameter, and assign the default in the body rather than in the parameter list (putting it in the parameter list makes it impossible for users to express "use the default" without explicitly encoding that default).